http/vibe/http/auth/digest_auth.d

```diff
@@ -56,7 +56,7 @@ class DigestAuthInfo
 		if (decoded.length != now.sizeof + secret.length) return NonceState.Invalid;
 		auto timebytes = decoded[0 .. now.sizeof];
 		auto time = () @trusted { return (cast(typeof(now)[])timebytes)[0]; } ();
-		if (timeout + time > now) return NonceState.Expired;
+		if (timeout + time < now) return NonceState.Expired;
```

Comment:
Looks like a second fix, which could itself use a test, but let's not delay this any longer.